### ISSUE TITLE
ffmpeg segmentation

### ISSUE THREAD

**Issue by oldwisenir** [fluent-ffmpeg/node-fluent-ffmpeg#569]
are there any events supporting ffmpeg capture segmentation of output files?
i would like to use some logic as a new segment was ended or started.

**Comment by njoyard** [comment]
No, there isn't, but maybe you can use the stderr event if ffmpeg outputs something when switching segments?

**Comment by oldwisenir** [comment]
there is no relevant info reguarding segmentation. thanks anyway

**Comment by njoyard** [comment]
Then you may be able to use an inotify lib (like chokidar or something) to get filesystem notifications when new segment files are created ?
